packages/connectors/connector-x/README.md

```diff
@@ -0,0 +1,71 @@
+# X connector
+
+The official Logto connector for X (formerly Twitter) social sign-in.
+
+**Table of contents**
+- [X connector](#x-connector)
+  - [Get started](#get-started)
+  - [Create an app in the X Developer Portal](#create-an-app-in-the-x-developer-portal)
+  - [Configure your connector](#configure-your-connector)
+    - [Config types](#config-types)
+  - [Test X connector](#test-x-connector)
+  - [Reference](#reference)
+
+## Get started
+
+The X connector enables end-users to sign in to your application using their own X (formerly Twitter) accounts via the X OAuth 2.0 authentication protocol.
+
+## Create an app in the X Developer Portal
+
+Go to the [X Developer Portal](https://developer.x.com/en/portal/projects-and-apps) and sign in with your X account. If you don’t have an account, you can register for one.
+
+Then, create an app.
+
+**Step 1:** Navigate to the App Creation Section.
+
+Once signed in, go to the "Projects & Apps" section and click on **"Create App"** (or **"New App"**, depending on the interface).
+
+**Step 2:** Fill in the App Details.
+
+Complete the form with the following information:
+
+- **App Name:** Provide a unique and descriptive name for your application.
+- **Application Description:** (Optional) Add a brief description of what your app does.
```

Comment:
```suggestion
- **App name:** Provide a unique and descriptive name for your application.
- **Application description:** (Optional) Add a brief description of what your app does.
```

Reply:
These are the original label from X developer page, I think it is better to keep the same.